dashboard/package.json

```diff
@@ -1,21 +1,40 @@
 {
-  "name": "pbench-dashboard-workflow",
+  "name": "pbench",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.17.8",
     "@patternfly/patternfly": "^4.183.1",
     "@patternfly/react-core": "^4.198.19",
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^0.26.1",
+    "@tippy.js/react": "^3.1.1",
+    "animate.css": "^4.1.1",
+    "axios": "^0.25.0",
```

Comment:
Do we really need to move axios back from 0.26.1 to 0.25.0? Why is that necessary?